tests/algorithms/test_fused_layernorm.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Tuple
+
+import pytest
+from torch.nn import LayerNorm
+
+from composer.algorithms.fused_layernorm import FusedLayerNorm, apply_fused_layernorm
+from composer.core.event import Event
+from composer.loggers import Logger
+from composer.models import BERTModel
+from tests.common import device
+from tests.fixtures.synthetic_hf_state import make_dataset_configs, synthetic_hf_state_maker
+
+
+@pytest.fixture()
+def synthetic_bert_state():
+    synthetic_config = make_dataset_configs(model_family=['bert'])[0]
+    return synthetic_hf_state_maker(synthetic_config)
+
+
+def assert_is_fln_instance(model: BERTModel):
+    pytest.importorskip('apex')
+    from apex.normalization.fused_layer_norm import FusedLayerNorm as APEXFusedLayerNorm
+
+    # ensure that within the entire model, no PyTorch LayerNorm exists, and at least one APEX FLN does.
+    found_apex_layernorm = False
+    for module_class in model.modules():
+        if isinstance(module_class, LayerNorm):
```

Comment:
nit: `Exception` and `BaseException` should never be raised directly; here in tests an assertion error is fine

```suggestion
        assert not isinstance(module_class, LayerNorm), 'A torch.nn.LayerNorm ...'
```